[quant][graphmode] torch.clamp

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #36994 [quant][graphmode] Test refactor -- extract graph checker
* #36889 [quant][graphmode] linear_relu
* **#36887 [quant][graphmode] torch.clamp**
* #36885 [quant][graphmode] fused conv3d + relu

Differential Revision: [D21110469](https://our.internmc.facebook.com/intern/diff/D21110469)

Commit:
[quant][graphmode] torch.clamp

[ghstack-poisoned]

Commit:
Update on "[quant][graphmode] torch.clamp"


Differential Revision: [D21110469](https://our.internmc.facebook.com/intern/diff/D21110469)

[ghstack-poisoned]

jerryzh168 commented on test/quantization/test_quantize_script.py:
```diff
@@ -1521,6 +1521,7 @@ def forward(self, x):
                 x = F.upsample(x, (32, 32))
                 x = F.upsample_bilinear(x, (32, 32))
                 x = F.upsample_nearest(x, (32, 32))
+                x = torch.clamp(x, -2, 2)
```
could you add tensor clamp as well, searching for clamp in pytorch doc shows:
torch.clamp (Python function, in torch)
torch.Tensor.clamp (Python method, in torch.Tensor)
torch.Tensor.clamp_ (Python method, in torch.Tensor)

Commit:
Update on "[quant][graphmode] torch.clamp"


Differential Revision: [D21110469](https://our.internmc.facebook.com/intern/diff/D21110469)

[ghstack-poisoned]

Commit:
Update on "[quant][graphmode] torch.clamp"


Differential Revision: [D21110469](https://our.internmc.facebook.com/intern/diff/D21110469)

[ghstack-poisoned]

Commit:
Update on "[quant][graphmode] torch.clamp"


Differential Revision: [D21110469](https://our.internmc.facebook.com/intern/diff/D21110469)

[ghstack-poisoned]
